recipes/djangocms-text-ckeditor/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "djangocms-text-ckeditor" %}
+{% set version = "5.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/djangocms-text-ckeditor-{{ version }}.tar.gz
+  sha256: eca45b3393879c61bb69d3c23df14a5fd4bef1f2ad66dc36a5bf7bfe06c6b7c3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - django-cms >=3.6
+    - html5lib >=1
+    - pillow
+    - packaging
+
+test:
+  imports:
+    - djangocms_text_ckeditor
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/django-cms/djangocms-text-ckeditor
+  summary: Text Plugin for django CMS with CKEditor support
+  license: BSD-3-Clause AND GPL-3.0 AND BSD-3-Clause AND MIT
```

You also need to define the GPL variant here.
```suggestion
  license: BSD-3-Clause AND GPL-3.0 AND MIT
```

---

https://github.com/django-cms/djangocms-text-ckeditor/blob/master/djangocms_text_ckeditor/static/djangocms_text_ckeditor/ckeditor/LICENSE.md
GNU General Public License Version 2 or later